src/treeChooser/treeChooser.controller.js

```diff
@@ -479,6 +487,15 @@ function TreeChooserController(
       .value();
   };
 
+  vm.chooseActive = function(item) {
+    var childs = item.getChildren();
```

Comment:
Children aren't necessarily leaf nodes. Use vm.itemsFlat to quickly find the first item that isShowing and isLeaf